### ISSUE TITLE
Is new code different from paper?

### ISSUE THREAD

**Issue by ofirkris** [FilipAndersson245/cartoon-gan#7]
Hello, the new results published look really good - well done!
How is the new approach different? is it just a new dataset?

**Comment by zimonitrome** [comment]
It is mainly the same approach as in the paper. The dataset is still the very same.

Our initial implementation in this repo had some errors. One such error was how we normalized and unnormalized images. This is not mentioned at all in the paper but has been corrected in the code.

Likewise, the paper does not mention any of the activation functions used at the very end of both the Discriminator and the Generator. Previously, the Discriminator used a Sigmoid function which has since been removed. This is because the training schemes now use mixed-precision mode which increases training speed and lower memory usage.

GANs can generally be (in my experience) very sensitive to randomness and learning rate. This most recent training used a different random seed than the original one and also did not use any scheduling. Previously we used something like this: https://www.jeremyjordan.me/nn-learning-rate/#stochasticgradientdescentwithwarmrestartssgdr

I assume that scheduling and mixed-precision are negligible. The difference in results could come from pure randomness and a different number of epochs trained.

We do however plan to improve this model.
